src/RollImage.cpp

```diff
@@ -779,15 +780,26 @@ void RollImage::analyzeMidiKeyMapping(void) {
 		}
 	}
 
+	//PMB temporary fix for Welte red rolls
```

Comment:
It's worth noting that we have yet to find an example among the 417 Welte red rolls now processed with this code for which this "temporary fix" has resulted in an incorrect tracker hole -> MIDI mapping. I'm still not entirely sure *why* it works, though. 😕

Reply:
Perhaps it would be appropriate to update this comment, then, do you think?  (I'm conscious that others will come along behind us on this code, most likely!).

Reply:
Yes, I think a "warning: problematic code" type of comment is still warranted here, but I'll change it to something more informative.